components/sup/doc/http_gateway_butterfly_schema.json

```diff
@@ -0,0 +1,483 @@
+                  },
+                  "votes": {
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        "update_counter": {
+          "type": "integer",
+          "description": "A count of how many times the election rumors have been updated",
+        }
+      }
+    },
+    "election_update": {
+      "type": "object",
+      "description": "A list of election update rumors",
+      "required": [
+        "list",
+        "update_counter"
+      ],
+      "properties": {
+        "list": {
+          "type": "object",
+          "description": "The election list",
+          "additionalProperties": {
+            "type": "object",
+            "description": "The name of a service group",
+            "properties": {
+              "election": {
+                "type": "object",
+                "description": "The election",
+                "properties": {
+                  "from_id": {
+                    "type": "string",
+                    "description": "The member ID being voted for"
+                  },
+                  "member_id": {
+                    "type": "string",
+                    "description": "The member ID being voted for"
+                  },
+                  "service_group": {
+                    "type": "string",
+                    "description": "The service group this election is running in"
+                  },
+                  "status": {
+                    "type": "string",
+                    "description": "The status of the election"
+                  },
+                  "suitability": {
+                    "type": "integer",
+                    "description": "How suitable this member is for election"
+                  },
+                  "term": {
+                    "type": "integer",
+                    "description": "The election term"
+                  },
+                  "votes": {
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        "update_counter": {
+          "type": "integer",
+          "description": "A count of how many times the election rumors have been updated",
+        }
+      }
+    },
+    "member": {
+      "type": "object",
+      "description": "The members of the butterfly network",
+      "required": [
+        "health",
+        "members",
+        "update_counter"
+      ],
+      "properties": {
+        "health": {
+          "type": "object",
+          "description": "The health of the members",
+          "additionalProperties": {
+            "type": "string",
+            "description": "The health of the member, e.g. Alive, Suspect, Confirmed, Departed"
```

Comment:
These possible values should be formally enumerated, instead of being part of a documentation string.

Reply:
Will do